pkg/log/log.go

```diff
@@ -77,13 +77,6 @@ func Entry() *logrus.Entry {
 	return logger
 }
 
-// FatalError provides an alternative to Entry().WithError(err).Fatal()
-// It supports error categorization and adds the error category as additional logging field
-// The error category can be set anywhere by just calling log.SetErrorCategory(category)
-func FatalError(err error, message string) {
-	Entry().WithError(err).WithField("category", GetErrorCategory()).Fatal(message)
-}
-
```

Comment:
Have you checked that this is not used somewhere outside this code-base?

Reply:
it just came in yesterday ... thus I am sure we can get rid of it again ...
I find it more convenient to stick with the default from the log ...

Reply:
from the `bufio/Scanner` docs

> Scanning stops unrecoverably at EOF, the first I/O error, or a token too large to fit in the buffer. When a scan stops, the reader may have advanced arbitrarily far past the last token. Programs that need more control over error handling or large tokens, or must run sequential scans on a reader, should use bufio.Reader instead.

From what I understand here the worst happening could be that the parsing is not 100% as accepted which I would accept ...